dependencies.yaml

```diff
@@ -160,6 +160,7 @@ worker_requires:
   optipng:
   sqlite3: '>= 3.24.0'   # for INSERT INTO … ON CONFLICT … DO UPDATE SET … required by cache service
   bsdtar:  # for extraction feature of OpenQA::Downloader
+  ntp-daemon:
```

Comment:
Souldn't this be a common dependency? Syncing just the worker's clock won't help if the web UI is still out of sync.